src/framework/config.ts

```diff
@@ -4,11 +4,12 @@ import path from 'path';
 
 import { IntegrationError } from '../errors';
 import {
+  ExecutionContext,
   GetStepStartStatesFunction,
   IntegrationInstanceConfigFieldMap,
   IntegrationInvocationConfig,
   IntegrationStep,
-  InvocationValidationFunction,
+  IntegrationInvocationValidationFunction,
```

Comment:
👍 Good rename here, matches the other `Integration*` things like `IntegrationInvocationConfig` etc.